stty: not buildable on Windows

Version: main branch
```
#cargo build -p uu_stty
   Compiling uu_stty v0.2.2 (Z:\src\uu\stty)
error[E0433]: failed to resolve: could not find `sys` in `nix`
  --> src\uu\stty\src\flags.rs:24:10
   |
24 | use nix::sys::termios::BaudRate;
   |          ^^^ could not find `sys` in `nix`

error[E0433]: failed to resolve: could not find `sys` in `nix`
  --> src\uu\stty\src\flags.rs:25:10
   |
25 | use nix::sys::termios::{
   |          ^^^ could not find `sys` in `nix`

error[E0433]: failed to resolve: could not find `sys` in `nix`
  --> src\uu\stty\src\stty.rs:21:10
   |
21 | use nix::sys::termios::{
   |          ^^^ could not find `sys` in `nix`

error[E0432]: unresolved import `nix::libc`
  --> src\uu\stty\src\stty.rs:20:10
   |
20 | use nix::libc::{O_NONBLOCK, TIOCGWINSZ, TIOCSWINSZ, c_ushort};
   |          ^^^^ could not find `libc` in `nix`

error[E0432]: unresolved imports `nix::ioctl_read_bad`, `nix::ioctl_write_ptr_bad`
  --> src\uu\stty\src\stty.rs:25:11
   |
25 | use nix::{ioctl_read_bad, ioctl_write_ptr_bad};
   |           ^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^^^ no `ioctl_write_ptr_bad` in the root
   |           |
   |           no `ioctl_read_bad` in the root

error[E0433]: failed to resolve: could not find `unix` in `os`
  --> src\uu\stty\src\stty.rs:30:14
   |
30 | use std::os::unix::fs::OpenOptionsExt;
   |              ^^^^ could not find `unix` in `os`
   |
note: found an item that was configured out
  --> D:\W\B\src\rustc-1.90.0-src\library\std\src\os\mod.rs:29:5
   |
   = note: the item is gated here
note: found an item that was configured out
  --> D:\W\B\src\rustc-1.90.0-src\library\std\src\os\mod.rs:84:41
   |
   = note: the item is gated here

error[E0433]: failed to resolve: could not find `unix` in `os`
  --> src\uu\stty\src\stty.rs:31:14
   |
31 | use std::os::unix::io::{AsRawFd, RawFd};
   |              ^^^^ could not find `unix` in `os`
   |
note: found an item that was configured out
  --> D:\W\B\src\rustc-1.90.0-src\library\std\src\os\mod.rs:29:5
   |
   = note: the item is gated here
note: found an item that was configured out
  --> D:\W\B\src\rustc-1.90.0-src\library\std\src\os\mod.rs:84:41
   |
   = note: the item is gated here

error[E0432]: unresolved import `std::os::fd`
  --> src\uu\stty\src\stty.rs:29:14
   |
29 | use std::os::fd::{AsFd, BorrowedFd};
   |              ^^ could not find `fd` in `os`
   |
note: found an item that was configured out
  --> D:\W\B\src\rustc-1.90.0-src\library\std\src\os\mod.rs:185:10
   |
   = note: the item is gated here

error[E0433]: failed to resolve: could not find `libc` in `nix`
   --> src\uu\stty\src\stty.rs:624:36
    |
624 | fn control_char_to_string(cc: nix::libc::cc_t) -> nix::Result<String> {
    |                                    ^^^^ could not find `libc` in `nix`

error[E0433]: failed to resolve: could not find `errno` in `nix`
   --> src\uu\stty\src\stty.rs:644:23
    |
644 |         _ => Err(nix::errno::Errno::ERANGE),
    |                       ^^^^^ could not find `errno` in `nix`

error[E0412]: cannot find type `Result` in crate `nix`
   --> src\uu\stty\src\stty.rs:484:67
    |
484 | fn print_special_setting(setting: &PrintSetting, fd: i32) -> nix::Result<()> {
    |                                                                   ^^^^^^ not found in `nix`
    |
help: consider importing one of these items
    |
 17 + use crate::io::Result;
    |
 17 + use std::fmt::Result;
    |
 17 + use std::io::Result;
    |
 17 + use std::result::Result;
    |
    = and 4 other candidates
help: if you import `Result`, refer to it directly
    |
484 - fn print_special_setting(setting: &PrintSetting, fd: i32) -> nix::Result<()> {
484 + fn print_special_setting(setting: &PrintSetting, fd: i32) -> Result<()> {
    |

error[E0412]: cannot find type `Result` in crate `nix`
   --> src\uu\stty\src\stty.rs:495:67
    |
495 | fn print_terminal_size(termios: &Termios, opts: &Options) -> nix::Result<()> {
    |                                                                   ^^^^^^ not found in `nix`
    |
help: consider importing one of these items
    |
 17 + use crate::io::Result;
    |
 17 + use std::fmt::Result;
    |
 17 + use std::io::Result;
    |
 17 + use std::result::Result;
    |
    = and 4 other candidates
help: if you import `Result`, refer to it directly
    |
495 - fn print_terminal_size(termios: &Termios, opts: &Options) -> nix::Result<()> {
495 + fn print_terminal_size(termios: &Termios, opts: &Options) -> Result<()> {
    |

error[E0412]: cannot find type `Result` in crate `nix`
   --> src\uu\stty\src\stty.rs:624:56
    |
624 | fn control_char_to_string(cc: nix::libc::cc_t) -> nix::Result<String> {
    |                                                        ^^^^^^ not found in `nix`
    |
help: consider importing one of these items
    |
 17 + use crate::io::Result;
    |
 17 + use std::fmt::Result;
    |
 17 + use std::io::Result;
    |
 17 + use std::result::Result;
    |
    = and 4 other candidates
help: if you import `Result`, refer to it directly
    |
624 - fn control_char_to_string(cc: nix::libc::cc_t) -> nix::Result<String> {
624 + fn control_char_to_string(cc: nix::libc::cc_t) -> Result<String> {
    |

error[E0412]: cannot find type `Result` in crate `nix`
   --> src\uu\stty\src\stty.rs:650:67
    |
650 | fn print_control_chars(termios: &Termios, opts: &Options) -> nix::Result<()> {
    |                                                                   ^^^^^^ not found in `nix`
    |
help: consider importing one of these items
    |
 17 + use crate::io::Result;
    |
 17 + use std::fmt::Result;
    |
 17 + use std::io::Result;
    |
 17 + use std::result::Result;
    |
    = and 4 other candidates
help: if you import `Result`, refer to it directly
    |
650 - fn print_control_chars(termios: &Termios, opts: &Options) -> nix::Result<()> {
650 + fn print_control_chars(termios: &Termios, opts: &Options) -> Result<()> {
    |

error[E0412]: cannot find type `Result` in crate `nix`
   --> src\uu\stty\src\stty.rs:700:62
    |
700 | fn print_settings(termios: &Termios, opts: &Options) -> nix::Result<()> {
    |                                                              ^^^^^^ not found in `nix`
    |
help: consider importing one of these items
    |
 17 + use crate::io::Result;
    |
 17 + use std::fmt::Result;
    |
 17 + use std::io::Result;
    |
 17 + use std::result::Result;
    |
    = and 4 other candidates
help: if you import `Result`, refer to it directly
    |
700 - fn print_settings(termios: &Termios, opts: &Options) -> nix::Result<()> {
700 + fn print_settings(termios: &Termios, opts: &Options) -> Result<()> {
    |

error[E0412]: cannot find type `Result` in crate `nix`
   --> src\uu\stty\src\stty.rs:801:11
    |
801 | ) -> nix::Result<()> {
    |           ^^^^^^ not found in `nix`
    |
help: consider importing one of these items
    |
 17 + use crate::io::Result;
    |
 17 + use std::fmt::Result;
    |
 17 + use std::io::Result;
    |
 17 + use std::result::Result;
    |
    = and 4 other candidates
help: if you import `Result`, refer to it directly
    |
801 - ) -> nix::Result<()> {
801 + ) -> Result<()> {
    |

error[E0425]: cannot find function `tiocgwinsz` in this scope
   --> src\uu\stty\src\stty.rs:488:22
    |
488 |             unsafe { tiocgwinsz(fd, &raw mut size)? };
    |                      ^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `tiocgwinsz` in this scope
   --> src\uu\stty\src\stty.rs:528:18
    |
528 |         unsafe { tiocgwinsz(opts.file.as_raw_fd(), &raw mut size)? };
    |                  ^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `tiocgwinsz` in this scope
   --> src\uu\stty\src\stty.rs:803:14
    |
803 |     unsafe { tiocgwinsz(fd, &raw mut size)? };
    |              ^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `tiocswinsz` in this scope
   --> src\uu\stty\src\stty.rs:815:14
    |
815 |     unsafe { tiocswinsz(fd, &raw mut size)? };
    |              ^^^^^^^^^^ not found in this scope

Some errors have detailed explanations: E0412, E0425, E0432, E0433.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `uu_stty` (lib) due to 20 previous errors
```

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -145,7 +145,6 @@ PROGS       := \
 	sleep \
 	sort \
 	split \
-	stty \
 	sum \
 	sync \
 	tac \
@@ -185,6 +184,7 @@ UNIX_PROGS := \
 	pinky \
 	stat \
 	stdbuf \
+	stty \
 	timeout \
 	touch \
 	tty \

